third_party/accessibility/ax/platform/ax_platform_node_base.cc

```diff
@@ -1764,6 +1764,91 @@ bool AXPlatformNodeBase::IsPlatformCheckable() const {
+    return;
+
+  const std::u16string& old_text = old_hypertext.hypertext;
+  const std::u16string& new_text = hypertext_.hypertext;
+
+  // TODO(accessibility) Plumb through which part of text changed so we don't
+  // have to guess what changed based on character differences. This can be
+  // wrong in some cases as follows:
+  // -- EDITABLE --
+  // If editable: when part of the text node changes, assume only that part
+  // changed, and not the entire thing. For example, if "car" changes to
+  // "cat", assume only 1 letter changed. This code compares common characters
+  // to guess what has changed.
+  // -- NOT EDITABLE --
+  // When part of the text changes, assume the entire node's text changed. For
+  // example, if "car" changes to "cat" then assume all 3 letters changed.
+  // Note, it is possible (though rare) that CharacterData methods are used to
+  // remove, insert, replace or append a substring.
+  bool allow_partial_text_node_changes =
+      GetData().HasState(ax::mojom::State::kEditable);
+  size_t prefix_index = 0;
+  size_t common_prefix = 0;
+  while (prefix_index < old_text.size() && prefix_index < new_text.size() &&
+         IsSameHypertextCharacter(old_hypertext, prefix_index, prefix_index)) {
+    ++prefix_index;
+    if (allow_partial_text_node_changes ||
+        (!IsText(old_text, prefix_index) && !IsText(new_text, prefix_index))) {
+      common_prefix = prefix_index;
+    }
+  }
+
+  size_t suffix_index = 0;
+  size_t common_suffix = 0;
+  while (common_prefix + suffix_index < old_text.size() &&
+         common_prefix + suffix_index < new_text.size() &&
+         IsSameHypertextCharacter(old_hypertext,
+                                  old_text.size() - suffix_index - 1,
+                                  new_text.size() - suffix_index - 1)) {
+    ++suffix_index;
+    if (allow_partial_text_node_changes ||
+        (!IsText(old_text, suffix_index, true) &&
+         !IsText(new_text, suffix_index, true))) {
+      common_suffix = suffix_index;
+    }
+  }
+
+  *start = common_prefix;
+  *old_len = old_text.size() - common_prefix - common_suffix;
+  *new_len = new_text.size() - common_prefix - common_suffix;
+}
+
+int AXPlatformNodeBase::FindTextBoundary(
+    ax::mojom::TextBoundary boundary,
+    int offset,
+    ax::mojom::MoveDirection direction,
+    ax::mojom::TextAffinity affinity) const {
+  if (boundary != ax::mojom::TextBoundary::kSentenceStart) {
+    std::optional<int> boundary_offset =
+        GetDelegate()->FindTextBoundary(boundary, offset, direction, affinity);
+    if (boundary_offset.has_value())
+      return *boundary_offset;
+  }
+
+  // std::vector<int32_t> unused_line_start_offsets;
+  // return static_cast<int>(
+  //     FindAccessibleTextBoundary(GetHypertext(), unused_line_start_offsets,
+  //                                boundary, offset, direction, affinity));
+  BASE_LOG() << "FindTextBoundary of SentenceStart is not yet supported.";
+  BASE_UNREACHABLE();
+  return offset;
```

Comment:
Is this an unnecessary feature?

Reply:
I'm not sure it's necessary or not. The `FindAccessibleTextBoundary` will refer to many dependencies which are exclude in flutter, and I checked the other code, the `ax::mojom::TextBoundary::kSentenceStart' is not supported in line 1617 of ax_position.h. 

```
      case ax::mojom::TextBoundary::kSentenceStart:
        BASE_LOG() << "Sentence boundaries are not yet supported.";
        BASE_UNREACHABLE();
        return CreateNullPosition();
```
I think we can uncomment this code in future when we got the scene to use it.

Reply:
Okay, thanks :)